### PR TITLE
(bug) user can't continue registration if stopping in middle

### PR DESCRIPTION
**Pull request by StanislavShevchenko**
# Description

added check CreateData in isDupUserData and added new test for this condition

About https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/1082

# How Has This Been Tested?

Start user registration, after entering the email, stop and try a new registration with the same parameters.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request ( for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/server/db/__tests__/UserPrivate.js [hunk 1]
```diff
@@ -60,6 +60,21 @@ describe('UserPrivate', () => {
     expect(user).not.toBeTruthy()
   })
 
+  it('Should not duplicate bu email', async () => {
```
Comment (sirpy) [inline src/server/db/__tests__/UserPrivate.js]:
```suggestion
  it('Should not duplicate by email', async () => {
```

File: src/server/db/__tests__/UserPrivate.js [hunk 2]
```diff
@@ -60,6 +60,21 @@ describe('UserPrivate', () => {
...
+    let isDupUser = await storage.isDupUserData({ email: testUser.email })
+    expect(isDupUser).not.toBeTruthy()
+  })
+
+  it('Should not duplicate bu email', async () => {
```
Comment (sirpy) [inline src/server/db/__tests__/UserPrivate.js]:
```suggestion
  it('Should not duplicate by email', async () => {
```

Comment (StanislavShevchenko) [reply src/server/db/__tests__/UserPrivate.js]: done

File: src/server/db/__tests__/UserPrivate.js (hunk 1 above)
Comment (StanislavShevchenko) [reply src/server/db/__tests__/UserPrivate.js]: done

**Pull request merged**